lib/aur-query

```diff
@@ -77,7 +77,7 @@ trap_exit() {
 }
 
 usage() {
-    printf 'usage: %s [-t [info|search] ] [-b by] [-aer] <package...>\n' "$argv0" >&2
+    printf 'usage: %s -t [info|search] [-b by] [-aer] <package...>\n' "$argv0" >&2
```

Comment:
`usage: info|search [...]` would imply `info` or `search` is required.  Does it make sense to just make one of the the default and return to `[-t info|search]`?

Reply:
I've thought of making `search` the default, but it's difficult to argue that either default makes more sense than the other one. For `aur-search`, the default is `search` (clearly).

Reply:
In that case a more usual synopsis might look like: `usage: aur query [-aer] [-b by] info|search package...`

Reply:
I've made some observations regarding default arguments which I think lead to an overall better design: https://github.com/AladW/aurutils/pull/944

With those changes, `aur query <args>` returns:
```
{"error":"No request type/data specified.","resultcount":0,"results":[],"type":"error","version":5}
```

Reply:
About positional arguments for the type, not sure about that, I've used `-t <type>` for a pretty long time. I guess it could be considered for further tools (`aur-sync--ninja` uses positional arguments exclusively.)